llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -764,11 +764,21 @@ define void @foo(ptr %arg0, ptr %arg1) {
   // Check create(InsertBefore)
   sandboxir::LoadInst *NewLd =
       sandboxir::LoadInst::create(Ld->getType(), Arg1, Align(8),
-                                  /*InsertBefore=*/Ret, Ctx, "NewLd");
+                                  /*InsertBefore=*/Ret, Ctx,
+                                  /*IsVolatile=*/false, "NewLd");
   EXPECT_EQ(NewLd->getType(), Ld->getType());
   EXPECT_EQ(NewLd->getPointerOperand(), Arg1);
   EXPECT_EQ(NewLd->getAlign(), 8);
   EXPECT_EQ(NewLd->getName(), "NewLd");
+
+  sandboxir::LoadInst *NewVld =
```

Comment:
Nit: The short version of NewVolatileLoad is probably better like this: `NewVLd` rather than `NewVld`

Reply:
should i replace the `Vld` with `VLd` or is it fine?

Reply:
That can be a separate NFC pull request.